buildenv/jenkins/jobs/infrastructure/copyrightCheckDir.groovy

```diff
@@ -0,0 +1,46 @@
+/*
+ * ===========================================================================
+ * (c) Copyright IBM Corp. 2019, 2019 All Rights Reserved
+ * ===========================================================================
+ * 
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  
+ *
+ * IBM designates this particular file as subject to the "Classpath" exception 
+ * as provided by IBM in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, see <http://www.gnu.org/licenses/>.
+ * 
+ * ===========================================================================
+*/
+
+def VERBOSE = ""
+def OUTPUT = ""
+
+timeout(time: 6, unit: 'HOURS') {
+    stage('Copyright Check') {
+        node ('worker') {
+            if ( params.ghprbGhRepository == "") {
+                echo "Repository to check not specified.  Rerun with the ghprbGhRepository parameter pointing to a valid git repository."
+                sh 'exit 1'
+            }
+            if ( params.Verbose == "1") {
+                VERBOSE="-v"
+            }
+            timestamps {
+                checkout scm
+                OUTPUT = sh (script: "sh buildenv/jenkins/jobs/infrastructure/copyrightCheckDir.sh REPO=${params.ghprbGhRepository}", returnStdout: true).trim()
+                echo OUTPUT
```

Comment:
The code is based on the code in the PR copyright checker and is my first pipeline script.
If the output is not collected and echo'ed, does it still make it to the console?
I found that stdout was not flushed as the script runs, and I ended up writing the output lines 'Processed nnnn files' to stderr, which is flushed as the script runs.
Is the observed lack of flushing just because the stderr is being captured in OUTPUT?  I'll do some more testing without it.